.astylerc

```diff
@@ -0,0 +1,34 @@
+# Mbed OS code style definition file for astyle
```

Comment:
Asking for clarification. Some of these flags and comments sound like they modify the files themselves. Is this actually the case?

Reply:
modify the files? can you elaborate?

Reply:
Do flags such as `convert-tabs` and `indent-cases` modify files?